apps/docs/src/routes/docs/theming/style-config-api.mdx

```diff
@@ -177,7 +177,7 @@ Now, let's create the `Alert` components that will consume our style config.
 
 ### Creating the components
 
-We create our components with the `createHopeComponent` function which add supports for the polymorphic `as` prop, style props and `sx`.
+We create our components with the `createHopeComponent function, which adds support for the polymorphic `as`prop, style props, and`sx`.
```

Comment:
Missing a ` at the end of createHopeComponent.

Reply:
I added one